qiita_pet/handlers/study_handlers/listing_handlers.py

```diff
@@ -42,14 +43,82 @@ def _get_shared_links_for_study(study):
     return ", ".join(shared)
 
 
-def _build_study_info(user, results=None):
-    """builds list of dicts for studies table, with all html formatted"""
+def _build_single_study_info(study, info):
```

Comment:
This auxuliary functions are missing the docstring. I think that putting the docstring is what will make the return structure more understandable, w/o the necessity to go over the code... (that's  also what I meant by breaking the functions...)